quiet.js

```diff
@@ -213,6 +243,14 @@ var Quiet = (function() {
                 addReadyCallback(opts.onReady);
             }
         }
+
+        var head = document.getElementsByTagName('head')[0];
```

Comment:
This is super gross and we should fix it, but I think it's one of the last remaining gross bits.